connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/UngracefulShutdownException.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!